src/app/pages/data-list/data-list.component.html

```diff
@@ -1,13 +1,13 @@
 <mat-toolbar color="primary">
     <div>Key Value List</div>
-    <div fxFlex style="margin:auto;"></div>
+    <div fxFlex style="margin: auto;"></div>
 </mat-toolbar>
 <mat-list>
-    <mat-list-item *ngFor="let item of list | keyValue;">
-        <div fxLayout="row" style="width:100%">
+    <mat-list-item *ngFor="let item of list | keyValue">
```

Comment:
pxb-info-list-item?

Reply:
I've updated this example to use info list item and found a style bug that only affects pxb-info-list-items used in ng v9.  Made a PR to patch the style.